src/utils/private.ts

```diff
@@ -0,0 +1,49 @@
+import _sodium from "libsodium-wrappers";
+import { getBotConfig, getBotContext } from "../bindings";
+import { Payload } from "../types";
+
+const CONFIG_REPO = "ubiquibot-config";
+const KEY_PATH = ".github/ubiquibot-config.yml";
+const KEY_NAME = "PSK";
+const KEY_PREFIX = "HSK_";
+
+export const getPrivateKey = async (): Promise<string | undefined> => {
+  try {
+    await _sodium.ready;
+    const sodium = _sodium;
+    const context = getBotContext();
+    const {
+      sodium: { publicKey, privateKey },
+    } = getBotConfig();
+    const payload = context.payload as Payload;
+
+    if (publicKey === "" || privateKey === "") {
+      return undefined;
+    }
+
+    const { data } = await context.octokit.rest.repos.getContent({
```

Comment:
Would you give me an example how `ubiquibot-config.yml` looks like? If there's some steps needed for integrators to create this file rather than putting raw private_key, we might need to update docs as well

Reply:
I've linked another PR for the wizard part,
the `ubiquibot-config.yml` should currently look like this:

```json
{
  "PSK": "<Encrypted_Private_Key_Value>"
}
```

Reply:
I've linked another PR for the wizard part,
the `ubiquibot-config.yml` should currently look like this:

```yml
  PSK: "<Encrypted_Private_Key_Value>"
```